tests/storage/migration.go

```diff
@@ -254,7 +256,8 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 			// Create dest PVC
 			switch mode {
 			case fsPVC:
-				libstorage.CreateFSPVC(destPVC, ns, size, nil)
+				// Add some overhead to the target PVC for filesystem.
+				libstorage.CreateFSPVC(destPVC, ns, sizeWithOverhead, nil)
```

Comment:
Is this test even interested in doing cross-volumemode migration? If not we can just adjust the source DV volume mode

Reply:
Not sure I get the comment, but yes, this table tests the combination with dv to pvc `fs->fs` and `fs ->block`

Reply:
Yeah I am a little confused as well now. This test now always does fs->fs for the `case fsPVC:` branch.
Is the extra overhead needed in such case? I guess it's needed since the source DV was inflated, and later we migrate into a vanilla PVC?

If that's the case, I guess that's fine, since if the user is exclusively using DVs then they don't have to worry about it?

Reply:
So this test covers fs dv to fs pvc and then we have [another](https://github.com/kubevirt/kubevirt/blob/main/tests/storage/migration.go#L278) for fs dv to fs dv. This might a bit redundant, but it might help us to catch some mismatches and differences between pure pvcs and dvs.

Reply:
Probably, this difference is due to the fact that on the destination PVC, it is kubevirt which creates the target image. If you think, this case isn't used by the user we can transform the PVC into a DV and remove the second test case

Reply:
We can keep it. It definitely has value.. once these tests are converted from serial->parallel it should be much of a hit to runtime

Reply:
KubeVirt CR update is the only reason to have these tests as serial. The fgs for volume migration should be GA soon (I hope), but then we still need to set the rollUpdateStrategy.